### PR TITLE
Send update instruction + Fix getBlockChainParameters

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Allow sending parameters updates with the gRPC client.

## Changes

- Added `sendUpdateInstruction` entrypoint to the gRPC client.
- Added missing fields to `getBlockChainParameters` response.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by soerenbf — CHANGES_REQUESTED** [review]
Tests (i.e. `nodejs/test`) fail with the addition to chain parameters due to type mismatch in `expectedJsons.ts`.

It would be nice if you could update the tests for `nodejs/test/client.test.ts` as well to ensure compatibility with both v1 and v2 api.

File: packages/common/src/types.ts [hunk 1]
```diff
@@ -2016,3 +2026,14 @@ export type SmartContractTypeValues =
...
+export type UpdateInstructionHeader = {
+    sequenceNumber: bigint;
+    effectiveTime: bigint;
+    timeout: bigint;
+};
+
+export type UpdateInstruction = {
+    header: UpdateInstructionHeader;
+    payload: HexString;
+};
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/common/src/types.ts]: Moved them :+1:

File: packages/common/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 ## 9.3.0
 
+### Fixed
+- Added missing fields to `getBlockChainParameters` response. (rootKeys, level1Keys, level2Keys)
+
 ### Added
 
+- `sendUpdateInstruction` to the gRPC Client.
```
Comment (soerenbf) [inline packages/common/CHANGELOG.md:10]: [the npm registry](https://www.npmjs.com/package/@concordium/common-sdk) states that 9.3.0 is already released?

**Review by soerenbf — APPROVED** [review]

File: packages/common/CHANGELOG.md (hunk 2 above)
Comment (shjortConcordium) [reply packages/common/CHANGELOG.md]: Yes, I think this is some faulty merging :thinking:
I've fixed it.

**Pull request merged**